### PR TITLE
Add bundlesize to budget size for largest bundles

### PR DESCRIPTION
**Pull request by zephraph**
`bundlesize` compares the _gzipped_ size of the file against a budget. I manually set these to be larger by a healthy margin then our individual bundle sizes. Should see a CI result comparing the sizes...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +58,44 @@
...
+    },
+    {
+      "path": "./public/assets/order2.*js",
+      "maxSize": "250 kB"
+    },
+    {
+      "path": "./public/assets/collect.*js",
+      "maxSize": "200 kB"
+    }
+  ],
```
Comment (damassi) [inline package.json]: Nice

File: webpack/envs/ciConfig.js [hunk 2]
```diff
@@ -1,21 +1,24 @@
 // @ts-check
 
+const fs = require("fs")
 const path = require("path")
 const stripAnsi = require("strip-ansi")
-const fs = require("fs").promises
+const { promisify } = require("util")
```
Comment (damassi) [inline webpack/envs/ciConfig.js]: ty 👍

File: yarn.lock [hunk 3]
```diff
@@ -1687,6 +1687,7 @@ antigravity@artsy/antigravity:
   version "0.1.3"
   resolved "https://codeload.github.com/artsy/antigravity/tar.gz/a4438d2fe9d0cdf71f1c47faba371cd3d004e140"
   dependencies:
+    coffeescript "1.11.1"
```
Comment (zephraph) [inline yarn.lock]: Why is this a thing. Stop being a thing. 

```suggestion
```

**Comment by alloy** [comment]
Awesome!

**Pull request merged**